Libraries/Experimental/SwipeableRow/SwipeableFlatList.js

```diff
@@ -35,7 +35,9 @@ type SwipableListProps = {
 type Props<ItemT> = SwipableListProps & FlatListProps<ItemT>;
 
 type State = {
-  openRowKey: ?string,
+  extraData: {
```

Comment:
why do we need extra object inside state? isn't it simpler keep it as before but put into flat list  `extraData={this.state}` ?

Reply:
@kesha-antonov 

Following the example in https://facebook.github.io/react-native/docs/flatlist.html

> By passing extraData={this.state} to FlatList we make sure FlatList itself will re-render when the state.selected changes. Without setting this prop, FlatList would not know it needs to re-render any items because it is also a PureComponent and the prop comparison will not show any changes.